lib/controllers/services.ts

```diff
@@ -123,3 +148,46 @@ export function GetService(serviceModel: ServiceModel): express.RequestHandler {
     )
   );
 }
+
+/**
+ * Returns the serviceId for all the Services that have sent
+ * at least one notification to the recipient with the provided fiscalCode.
+ */
+export function GetSenderServicesHandler(
+  senderServiceModel: SenderServiceModel
+): IGetSenderServicesHandler {
+  return async (_, __, ___, fiscalCode) => {
+    const retrievedServicesIterator = senderServiceModel.findSenderServicesForRecipient(
+      fiscalCode
+    );
+    const senderServicesIterator = mapResultIterator(
+      retrievedServicesIterator,
+      service => pick(["serviceId"], service)
+    );
+    return ResponseSuccessJsonIterator(senderServicesIterator);
+  };
+}
+
+/**
+ * Wraps a GetSenderServices handler inside an Express request handler.
+ */
+export function GetSenderServices(
+  serviceModel: ServiceModel,
+  senderServiceModel: SenderServiceModel
+): express.RequestHandler {
+  const handler = GetSenderServicesHandler(senderServiceModel);
+  const azureUserAttributesMiddleware = AzureUserAttributesMiddleware(
+    serviceModel
+  );
+  const middlewaresWrap = withRequestMiddlewares(
+    AzureApiAuthMiddleware(new Set([UserGroup.ApiFullProfileRead])),
```

Comment:
we need a new API permission for this operation, something like `ApiServiceByRecipientQuery`